content/en/blog/2024/otel-docs-survey/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Insights from the OpenTelemetry Docs Usability Survey
+linkTitle: OpenTelemetry Docs Survey
+date: 2024-12-13
+author: '[Tiffany Hrabusa](https://github.com/tiffany76) (Grafana Labs)'
+issue: https://github.com/open-telemetry/opentelemetry.io/issues/5793
+sig: Communications, End-User
+# prettier-ignore
+cSpell:ignore: Hrabusa
+---
+
+The OpenTelemetry End-User SIG recently surveyed the community to find out how
+user-friendly OpenTelemetry’s documentation is. In an earlier survey, two-thirds
+of respondents named comprehensive documentation as a top resource they wished
+they’d had when getting started with OpenTelemetry. So we decided to dig a
+little deeper. The Docs Usability Survey asked users where they go for OTel
+documentation, what they’d like to see more of in the docs, and how they rate
+the current state of the docs. We received 48 responses, which we'll use to
+focus our documentation efforts and help us improve in key areas.
+
+Check out the results!
+
+## Key takeaways
+
+- Folks want more visual aids like diagrams and screenshots.
+- Of the three types of documentation we asked about (component concepts,
+  installation instructions, and troubleshooting), the troubleshooting docs need
+  the most work.
+- When asked what information they’d most like to see added to OTel’s docs, the
+  top answers were more examples and more coverage, both in depth and breadth.
+- The Collector docs were the most consulted docs set based on number of
+  ratings.
+- The Java documentation was the overall highest rated docs, after normalization
```

Comment:
```suggestion
- The [Java documentation](/docs/languages/java/) was the overall highest rated docs, after normalization
```

looks like the work put into it is honored, cc @zeitlinger @jeanbisutti @jack-berg @trask @jaydeluca

Reply:
Very nice! Might again be worth cross-referencing our two posts.